### DISCUSSION TITLE
I Heart Movies

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#656] [Q&A]
**Channel Name:** I Heart Movies
**Channel ID:** IHeartMovies.ph
**Country:** Philippines
**Language:** Tagalog
**External Link (optional):** xxx
**Notes (optional):** Free to air movie channel in the Philippines.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19657
